### ISSUE TITLE
x_create_picture_with_pictfmt_and_pixmap ERROR, root_damaged ERROR

### ISSUE THREAD

**Issue by jakariyaa** [yshui/compton#1159]
### Platform
System:
  Kernel: 5.15.0-91-generic x86_64 bits: 64 compiler: gcc v: 11.4.0 Desktop: Xfce 4.18.1
    tk: Gtk 3.24.33 wm: xfwm dm: LightDM Distro: Linux Mint 21.2 Victoria base: Ubuntu 22.04 jammy

### GPU, drivers, and screen setup
Graphics:
<details>

  ```
  Device-1: AMD Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] vendor: Gigabyte
    driver: amdgpu v: kernel pcie: speed: 8 GT/s lanes: 16 ports: active: HDMI-A-1
    empty: DP-1, DP-2, DP-3, DVI-D-1 bus-ID: 06:00.0 chip-ID: 1002:67df
  Display: x11 server: X.Org v: 1.21.1.4 compositor: xfwm v: 4.18.0 driver: X:
    loaded: amdgpu,ati unloaded: fbdev,modesetting,vesa gpu: amdgpu display-ID: :0.0 screens: 1
  Screen-1: 0 s-res: 1920x1080 s-dpi: 96
  Monitor-1: HDMI-A-0 mapped: HDMI-A-1 model: ViewSonic VA2215-FHD res: 1920x1080 dpi: 102
    diag: 545mm (21.5")
  OpenGL: renderer: AMD Radeon RX 580 Series (polaris10 LLVM 15.0.7 DRM 3.42 5.15.0-91-generic)
    v: 4.6 Mesa 23.0.4-0ubuntu1\~22.04.1 direct render: Yes
  ```

</details>
    glxinfo -B   
<details>   

```
name of display: :0.0
display: :0  screen: 0
direct rendering: Yes
Extended renderer info (GLX_MESA_query_renderer):
    Vendor: AMD (0x1002)
    Device: AMD Radeon RX 580 Series (polaris10, LLVM 15.0.7, DRM 3.42, 5.15.0-91-generic) (0x67df)
    Version: 23.0.4
    Accelerated: yes
    Video memory: 8192MB
    Unified memory: no
    Preferred profile: core (0x1)
    Max core profile version: 4.6
    Max compat profile version: 4.6
    Max GLES1 profile version: 1.1
    Max GLES[23] profile version: 3.2
Memory info (GL_ATI_meminfo):
    VBO free memory - total: 7889 MB, largest block: 7889 MB
    VBO free aux. memory - total: 5849 MB, largest block: 5849 MB
    Texture free memory - total: 7889 MB, largest block: 7889 MB
    Texture free aux. memory - total: 5849 MB, largest block: 5849 MB
    Renderbuffer free memory - total: 7889 MB, largest block: 7889 MB
    Renderbuffer free aux. memory - total: 5849 MB, largest block: 5849 MB
Memory info (GL_NVX_gpu_memory_info):
    Dedicated video memory: 8192 MB
    Total available memory: 14082 MB
    Currently available dedicated video memory: 7889 MB
OpenGL vendor string: AMD
OpenGL renderer string: AMD Radeon RX 580 Series (polaris10, LLVM 15.0.7, DRM 3.42, 5.15.0-91-generic)
OpenGL core profile version string: 4.6 (Core Profile) Mesa 23.0.4-0ubuntu1~22.04.1
OpenGL core profile shading language version string: 4.60
OpenGL core profile context flags: (none)
OpenGL core profile profile mask: core profile

OpenGL version string: 4.6 (Compatibility Profile) Mesa 23.0.4-0ubuntu1~22.04.1
OpenGL shading language version string: 4.60
OpenGL context flags: (none)
OpenGL profile mask: compatibility profile

OpenGL ES profile version string: OpenGL ES 3.2 Mesa 23.0.4-0ubuntu1~22.04.1
OpenGL ES profile shading language version string: OpenGL ES GLSL ES 3.20
```

</details>

### picom version
picom --version
vgit-49645

picom --diagnostics
**Version:** vgit-49645

### ##### 
* Shape: Yes
* RandR: Yes
* Present: Present

##### Misc:

* Use Overlay: Yes
* Config file used: None

##### Drivers (inaccurate):

AMDGPU, Radeon

##### Backend: glx

* Driver vendors:
 * GLX: Mesa Project and SGI
 * GL: AMD
* GL renderer: AMD Radeon RX 580 Series (polaris10, LLVM 15.0.7, DRM 3.42, 5.15.0-91-generic)
* Accelerated: 1

##### Backend: egl

* Driver vendors:
 * EGL: Mesa Project
 * EGL driver: radeonsi
 * GL: AMD
* GL renderer: AMD Radeon RX 580 Series (polaris10, LLVM 15.0.7, DRM 3.42, 5.15.0-91-generic)

### Configuration:

// Did not set any just installed picom.

### Steps of reproduction

1. Just run **picom** command in terminal (with or without **sudo**)

### Expected behavior

No Error after running picom

### Current Behavior

After running picom command:

[ 12/28/2023 17:53:38.901 x_create_picture_with_pictfmt_and_pixmap ERROR ] failed to create picture (X error 8 MATCH request 139 minor 4 serial 2676)
[ 12/28/2023 17:53:38.901 root_damaged ERROR ] Failed to bind root back pixmap

**Comment by hossine7** [comment]
same here

**Comment by absolutelynothelix** [comment]
it’s an issue with xfce and from my experience it’s harmless. see #984 for details.

**Comment by hossine7** [comment]
so we can just ignore it then

**Comment by jakariyaa** [comment]
> it’s an issue with xfce and from my experience it’s harmless. see #984 for details.

Well. That's that then. If it is "harmless" and there's no fix for now, I think I can conclude this as resolved and close this issue? Anyone has anything to add?

**Comment by absolutelynothelix** [comment]
let’s keep this open as it’s a bug that needs to be fixed and there is no issue for it afaik.

**Comment by absolutelynothelix** [comment]
closing as fixed in #984.
